tests/modules/nf-core/coreograph/main.nf

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { COREOGRAPH } from '../../../../modules/nf-core/coreograph/main.nf'
+
+workflow test_coreograph {
+
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        file('/workspace/modules/tests/modules/nf-core/coreograph/TestdataCoreographHighRes.tif', checkIfExists: true)
```

Comment:
Once https://github.com/nf-core/modules/pull/3199 merges (in queue) this can point towards the new test data
```
file(params.test_data['imaging']['core_detection']['image'], checkIfExists: true)
```

Reply:
https://github.com/nf-core/modules/pull/3199 merged! You should be good to switch to the central test-dataset and delete the local test files you added.

Reply:
switched to the central set and deleted the other one!